h2o-docs/src/product/faq/general-troubleshooting.rst

```diff
@@ -130,14 +130,22 @@ Depending on the user-specified predictors, max_depth per tree, the number of tr
 
 **What's the best approach to help diagnose a possible memory problem on a cluster?**
 
-We've found that the best way to understand JVM memory consumption is to turn on the following JVM flags:
+We've found that the best way to understand JVM memory consumption is to turn on specific flags. These flags differ si
+
+For Java Java version < 10, the following flags are available:
 
 ::
 
    -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCTimeStamps
 
 You can then use the following tool to analyze the output: http://www.tagtraum.com/gcviewer-download.html
```

Comment:
This tool does not support the new logging standard yet (see [issues on GitHub](https://github.com/chewiebug/GCViewer/issues)). That is why I moved it here.